public/swagger.v1.json

```diff
@@ -969,15 +969,12 @@
           "x-go-name": "OpenIssues"
         },
         "owner": {
-          "x-go-name": "Owner",
           "$ref": "#/definitions/User"
```

Comment:
I don't know the implications of the changes to this file, so I'll defer to @ethantkoenig here.